django_select2/forms.py

```diff
@@ -182,10 +182,16 @@ def __init__(self, **kwargs):
         Args:
             data_view (str): URL pattern name
             data_url (str): URL
-
+            dependent_fields (List[str]): List of dependent parent fields.
+                The value of the dependent field will be passed as to :func:`.filter_queryset`.
+                It can be used to further restrict the search results. For example, a city
+                widget could be dependent on a country.
         """
         self.data_view = kwargs.pop('data_view', None)
         self.data_url = kwargs.pop('data_url', None)
+        self.dependent_fields = kwargs.pop('dependent_fields', {})
+        if isinstance(self.dependent_fields, tuple) or isinstance(self.dependent_fields, list):
```

Comment:
should be:
```python
isinstance(self.dependent_fields, (tuple, list))
```
But don't type check. There should be only one way to use this.

`dependent_fields` should be a dict (or a list or tuples). BTW, if would be nice if one could set this as a class attribute too.